tpl/tplimpl/embedded/templates/_default/rss.xml

```diff
@@ -21,7 +21,7 @@
     <managingEditor>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</managingEditor>{{end}}{{ with .Site.Author.email }}
     <webMaster>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</webMaster>{{end}}{{ with .Site.Copyright }}
     <copyright>{{.}}</copyright>{{end}}{{ if not .Date.IsZero }}
-    <lastBuildDate>{{ .Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</lastBuildDate>{{ end }}
```

Comment:
Hugo can have RSS feed on many levels: top level, per section, per category (taxonomy); I thinks this date should best reflect the last modification date of the feed itself, which is `.Lastmod`.

Reply:
Simply `.Lastmod`  doesn't work, my previous commit would get that behavior calculated:
 https://github.com/gohugoio/hugo/commit/22463e919fa9ef76f1ae40d793b4262b75435bc2

Should I revert to that?

Reply:
If `Lastmod` doesnt't do what you want, we can certainly discuss that, but you need to bring it up in another issue.